kiwi/system/root_init.py

```diff
@@ -84,6 +85,8 @@ def create(self):
             data.sync_data(
                 options=['-a', '--ignore-existing']
             )
+            if Defaults.is_buildservice_worker():
+                copy('/.buildenv', self.root_dir)
```

Comment:
Sure @schaefi, no problem, I can revert it.

About the get_buildservice_env_name I like the idea, something similar I have on the way for other kiwi files in #423 (I did not make the PR yet because this one goes first).